feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -25,15 +27,11 @@ import io.github.droidkaigi.confsched2020.di.PageScope
 import io.github.droidkaigi.confsched2020.ext.assistedActivityViewModels
 import io.github.droidkaigi.confsched2020.ext.assistedViewModels
 import io.github.droidkaigi.confsched2020.ext.getThemeColor
-import io.github.droidkaigi.confsched2020.model.Session
-import io.github.droidkaigi.confsched2020.model.Speaker
-import io.github.droidkaigi.confsched2020.model.SpeechSession
-import io.github.droidkaigi.confsched2020.model.defaultLang
-import io.github.droidkaigi.confsched2020.model.defaultTimeZoneOffset
```

Comment:
Thank you 🙏
Probably your Project.xml is initialized🧐
We don't use star import by Android Kotlin Style Guide

Reply:
Thank you for the review.
Corrected and pushed.